tempodb/tempodb.go

```diff
@@ -142,17 +143,20 @@ func New(cfg *Config, logger log.Logger) (Reader, Writer, Compactor, error) {
 		return nil, nil, nil, err
 	}
 
-	if cfg.Memcached != nil {
-		r, w, err = memcached.New(r, w, cfg.Memcached, logger)
+	var cacheBackend cache.Client
+	var defaultCache = false
 
-		if err != nil {
-			return nil, nil, nil, err
-		}
+	switch cfg.Cache {
```

Comment:
i always thought it was funny you could cache in both redis and memcached if you wanted :).  no issues with this change.

Reply:
x2 power? :D